spec/models/user_spec.rb

```diff
@@ -0,0 +1,16 @@
+require 'spec_helper'
+
+describe User do
+    it 'should have email, email' do
```

Comment:
This message does not make sense, should have email, email? Also do not use `should` in messages. Read more here -> http://betterspecs.org/#should